src/api.ts

```diff
@@ -382,11 +398,16 @@ export class MirrordAPI {
   *                             `undefined` if we found the target in the config,
   *                             and the widget was not shown.
   */
-  async binaryExecute(quickPickSelection: UserSelection | undefined, configFile: string | null, executable: string | null, configEnv: EnvVars): Promise<MirrordExecution> {
+  async binaryExecute(quickPickSelection: UserSelection | undefined, configFile: string | null, executable: string | null, configEnv: EnvVars, workspacePath: string | undefined): Promise<MirrordExecution> {
     tickMirrordForTeamsCounter();
     tickFeedbackCounter();
     tickDiscordCounter();
 
+    let branchName: string | void;
+    if (workspacePath !== null && typeof workspacePath === 'string') {
```

Comment:
```suggestion
    if (workspacePath !== undefined) {
```
It should be either `undefined` or a `string`, given the function signature, no? :eyes: